src/ruby/lib/grpc/errors.rb

```diff
@@ -57,9 +59,6 @@ def to_status
     #
     # @return [Google::Rpc::Status, nil]
     def to_rpc_status
-      # Lazily require google_rpc_status_utils to scope
-      # loading protobuf_c.so to the users of this method.
-      require_relative './google_rpc_status_utils'
```

Comment:
nit: it's outside of the scope of this PR, but while we're here, can we also remove the two lines below here: I believe they are dead/unused code.

E.g. this could be change to just `GoogleRpcStatusUtils.extract_google_rpc_status(to_status)`

Reply:
Yes. I've made this change and the end2end script continues to pass.